### PR TITLE
Update intro text with link to license, research team and Github

### PR DESCRIPTION
**Pull request by Rensvandeschoot**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -6,6 +6,12 @@ Welcome to the documentation of ASReview! It includes an :doc:`installation
 </intro/installation>` guide, a :doc:`quick tour </lab/oracle>`, details how
 to prepare your :doc:`dataset </intro/datasets>` and much more - enjoy!
 
+ASReview is `open source<https://github.com/asreview/asreview>`_ research
```
Comment (J535D165) [inline docs/source/index.rst:9]: Missing whitespace between `open source` and `<`.

File: docs/source/index.rst [hunk 2]
```diff
@@ -6,6 +6,12 @@ Welcome to the documentation of ASReview! It includes an :doc:`installation
...
+software developed by
+`researchers<https://asreview.readthedocs.io/en/latest/intro/zen.html#research-team>`_
```
Comment (J535D165) [inline docs/source/index.rst:11]: Please use 
```
:ref:`researchers <intro/zen:research team>`
```
such that it renders nicely in the PDF.

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks. Two minor remarks on the syntax.

File: docs/source/index.rst (hunk 2 above)
Comment (Rensvandeschoot) [reply docs/source/index.rst]: Thanks! text has been updated

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks!

**Pull request merged**